atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -160,6 +160,10 @@ def get_cachito_session(workflow):
     return CachitoAPI(config['api_url'], **api_kwargs)
 
 
+def get_allow_multiple_remote_sources(workflow):
```

Comment:
Should we have `fallback` here like other methods have?

Reply:
we don't need explicit fallback, that was used before, but we don't need it for new methods

Reply:
ok